packages/components/src/accordion/tests/jest/Accordion.ssr.test.tsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * @jest-environment node
+ */
+import { Accordion } from "@components/accordion";
+import { Content } from "@components/placeholders";
+import { H3 } from "@components/typography";
+import { Item } from "@components/collection";
+import { renderToString } from "react-dom/server";
+
+test("Server-side rendering: renders on a server without crashing", () => {
```

Comment:
Please follow a naming convention similar to the other Jest tests. Should be something like "can render on the server"